ui/less/ad_controls.less

```diff
@@ -71,6 +71,10 @@
   .ad-text-shadow();
 }
 
+.shaka-ad-controls .shaka-ad-position:not(.shaka-hidden) + .shaka-ad-counter {
```

Comment:
Hm. I tried this out in practice, and I feel like 5 px isn't enough. 10 or maybe even 15 px might be better.
Looks good otherwise.

Reply:
Well, I put the value you want, if you want 10 I put it at 10.

Reply:
I would suggest we not use pixels at all.  Spacing between two text elements should probably be based on the font size.  I think something in the range of 0.5em to 1.0em looks right, and will scale with changes to the font.

Reply:
Done